Azure-Samples/azure-search-knowledge-mining
# Vcurtis/bubble interactivemap

## Purpose
Applies UI change to interactive map enabling automatic bubble clustering of many crowded elements. 

## Pull Request Type
What kind of change does this Pull Request introduce?
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Test by running the search ui and zoom in and out of the interactive map to see changes. Initial commit. Ensure existing functionality still works.


## What to Check
![thumbnail_image002](https://user-images.githubusercontent.com/62909651/80827410-c2efb280-8b98-11ea-97f2-9f4fd0d214d0.png)
Verify that the following are valid
*bubble clustering


## Other Information
